kvdb/etcd/config.go

```diff
@@ -25,4 +25,8 @@ type Config struct {
 	InsecureSkipVerify bool `long:"insecure_skip_verify" description:"Whether we intend to skip TLS verification"`
 
 	CollectStats bool `long:"collect_stats" description:"Whether to collect etcd commit stats."`
+
+	// SingleWriter should be set to true if we intend to only allow a
+	// single writer to the database at a time.
+	SingleWriter bool
```

Comment:
Is this going to be surfaced/set in a follow up PR? As is looks like it's only used for `NewBackend` on `EtcdTestFixture`?

Reply:
Yes this is going to be used in the fully-remote DB PR to allow legacy code that assumes single writer to run correctly. (Wallet)